src/core/EffectComposer.vue

```diff
@@ -111,6 +111,11 @@ onLoop(({ delta }) => {
 onUnmounted(() => {
   effectComposer.value?.dispose()
 })
+
+onMounted(() => {
+  if (!sizes.height.value && !sizes.width.value) return
+  if (props.enabled) initEffectComposer()
```

Comment:
```suggestion
  if (sizes.height.value && sizes.width.value && props.enabled) {
    initEffectComposer()
    stop()
  }
```

Caution: I did not test the suggestion 😄